code/modules/clothing/head/helmet.dm

```diff
@@ -277,6 +278,13 @@
 	name = "\improper M10 corpsman helmet"
 	flags_item_map_variant = (ITEM_JUNGLE_VARIANT|ITEM_ICE_VARIANT|ITEM_ICE_PROTECTION)
 
+/obj/item/clothing/head/helmet/marine/heavy
+	name = "\improper M10H pattern marine helmet"
+	desc = "A standard M10H Pattern Helmet. It reads on the label, 'The difference between an open-casket and closed-casket funeral. Wear on head for best results.'."
+	icon_state = "heavyhelmet"
+	armor = list("melee" = 75, "bullet" = 70, "laser" = 50, "energy" = 20, "bomb" = 45, "bio" = 0, "rad" = 0, "fire" = 20, "acid" = 45)
+	accuracy_mod = -0.1
```

Comment:
Just saying, but this is a % chance to miss. You are adding 0.1% chance to miss. Might as well add nothing.